docs/docs/create-apps/general-app.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+---
+
+# General instructions
+
+You can create and deploy apps for multiple frameworks with the following steps.
+
+## Pre-requisites
+
+Before deploying/launching your app with JHub Apps Launcher, you'll need:
+
+- working code in a Jupyter Notebook or a Python script, and
+- environment with all the necessary packages.
+
+:::note
+For all frameworks, you need to include the `jhsingle-native-proxy` package in your environment along with the framework itself (e.g., bokeh, panel, etc.)
+
```

Comment:
```suggestion
Also, be aware that because `jhub-apps` may be launching your app from a different directory than you used during development, its important to always define absolute paths in your apps. For example, if you use `pathlib` to define a path, you'll need to call `.resolve()` in order to make it absolute. 
```